.github/workflows/pr-build.yaml

```diff
@@ -2,19 +2,19 @@ name: Go Build
 
 on:
   push:
-    branches: [ main, development, feat/* ]
+    branches: [ main, rc/*, feat/* ]
   pull_request:
-    branches: [ main, development, feat/* ]
+    branches: [ main, rc/*, feat/* ]
 
 jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
     steps:
-      - name: Set up Go 1.x
+      - name: Set up Go 1.17.6
         uses: actions/setup-go@v2
         with:
-          go-version: ^1.15.6
+          go-version: ^1.17.6
```

Comment:
remove `^` as current libs will fail if go1.19 is used for example

Reply:
removed